renv.lock

```diff
@@ -1335,9 +1335,9 @@
     },
     "tibble": {
       "Package": "tibble",
-      "Version": "3.1.8",
+      "Version": "3.2.1",
```

Comment:
The model API was returning weird errors originating from a `vectbl_assign` function reporting that `"DLL requires the use of native symbols"`; I googled around and folks suggested [updating `tibble`](https://stackoverflow.com/a/76314441), which worked.